admin/services/table-name-service.js

```diff
@@ -0,0 +1,51 @@
+'use strict'
+const config = require('../config')
+const R = require('ramda')
+
+const NAMES = {
+  PREPARED_CHECK: 'preparedCheck'
+}
+
+Object.freeze(NAMES)
+
+const nameService = {
+  /**
+   * Returns the correct table name for the environment based on a key.
+   * E.g.
+   * @param tableName - the basic queue name
+   * @return {string}
+   */
+  getName: function (tableName) {
+    const prefix = config.Azure.tablePrefix
+
+    const lookupKey = getKeyFromValue(tableName)
+
+    if (!NAMES.hasOwnProperty(lookupKey)) {
+      throw new Error('Table name not found: ' + tableName)
+    }
+
+    // If the config prefix is an empty string - there is nothing to do, so return the queue name as it is.
+    if (!prefix) {
+      return tableName
+    }
+
+    // Queue naming info: https://docs.microsoft.com/en-us/rest/api/storageservices/naming-queues-and-metadata
+    const stem = capitalise(tableName)
```

Comment:
I read the doc and if I understand correctly on number 3 it states 'all letters in a queue name must be lowercase'.
In your capitalise function your first param will be uppercased and the rest will be lower cased.
Am I missing something?